privacyidea/translations/de/LC_MESSAGES/messages.po

```diff
@@ -67,634 +67,658 @@ msgstr "Ungültige Seriennummer."
 
-#: lib/policy.py:1771
+#: lib/policy.py:1779
 msgid "This is the default token type in the token enrollment dialog."
 msgstr "Die Voreinstellung des Tokentyps im Rollout-Dialog."
 
-#: lib/policy.py:1777
+#: lib/policy.py:1785
 msgid "A list of realm names, which are displayed in a drop down menu in the WebUI login screen. Realms are separated by white spaces."
 msgstr "Eine durch Leerzeichen getrennte Liste von Realmnamen, die in einer Auswahlliste beim Login in das Webinterface angezeigt werden."
 
-#: lib/policy.py:1783
+#: lib/policy.py:1791
 msgid "If this checked, the administrator will not see the welcome dialog anymore."
 msgstr "Dem Administrator wird der Willkommen-Dialog nicht mehr angezeigt."
 
-#: lib/policy.py:1788
+#: lib/policy.py:1796
 msgid "Per default disabled actions result in disabled buttons. When checking this action, buttons of disabled actions are hidden."
 msgstr "Buttons von Funktionen, die ein Benutzer nicht ausführen darf, werden ausgeblendet. (Das Standardverhalten ist, dass die Buttons deaktiviert werden)"
 
-#: lib/policy.py:1793
+#: lib/policy.py:1801
 msgid "If this is checked, the seed will be displayed as text during enrollment."
 msgstr "Wenn dies angehakt ist, wird das Seed während des Rollouts als Text angezeigt."
 
-#: lib/radiusserver.py:200 lib/radiusserver.py:231
+#: lib/radiusserver.py:204 lib/radiusserver.py:236
 msgid "The RADIUS secret is too long"
 msgstr "Das RADIUS Secret ist zu lang"
 
-#: lib/token.py:2308
+#: lib/token.py:2311
 msgid "Admin is allowed to initalize {0!s} tokens."
 msgstr "Der Administrator kann %s Token ausrollen."
 
-#: lib/token.py:2316
+#: lib/token.py:2319
 msgid "The user is allowed to enroll a {0!s} token."
 msgstr "Der Benutzer darf einen %s Token ausrollen."
 
-#: lib/token.py:2347
+#: lib/token.py:2350
 msgid "Set the maximum allowed PIN length of the {0!s} token."
 msgstr "Setze die maximal erlaubte PIN-Länge für den {0!s} Token."
 
-#: lib/token.py:2354
+#: lib/token.py:2357
 msgid "Set the minimum required PIN length of the {0!s} token."
 msgstr "Sete die minimal notwendite PIN-Länge für den {0!s} Token."
 
-#: lib/token.py:2360
+#: lib/token.py:2363
 msgid "Specifiy the required PIN contents of the {0!s} token. (c)haracters, (n)umeric, (s)pecial, (o)thers. [+/-]!"
 msgstr "Definiere den notwendigen Inhalte der OTP PIN für den {0!s} Token. (c) Buchstaben, (n) Ziffern, (s) Sonderzeichen, (o) Andere [+/-]!"
 
-#: lib/tokenclass.py:1469
+#: lib/tokenclass.py:1454
 msgid "please enter otp: "
 msgstr "Bitte geben Sie einen OTP-Wert ein: "
 
-#: lib/eventhandler/base.py:124
+#: lib/eventhandler/base.py:132
 msgid "The user realm, for which this event should apply."
 msgstr "Benutzer-Realm, für den das Event greifen soll."
 
-#: lib/eventhandler/base.py:129
+#: lib/eventhandler/base.py:137
+msgid "The user resolver, for which this event should apply."
+msgstr "Der Resolver des Benutzers, für den diese Ereignis-Definition greift."
```

Comment:
The user resolver = Der Benutzerresolver
The user's resolver = Der Resolver des Benutzers
maybe add the 's or change the translation to preempt ambiguity

Reply:
Thats true, I will change it to the "resolver of the user".
Especially since there is no "token resolver"; but "the resolver of the token" makes perfect sense.

Reply:
done